### PR TITLE
Logs Data Platform subscription configuration

### PR DESCRIPTION
**Pull request by Heiiji**
LDP subscription configuration on manager
| Question         | Answer
| ---------------- | ---
| Branch?          | master
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-5660
| License          | BSD 3-Clause

## Description

Please do not pay attention to the two first commits.
They basically re-apply (reverts) changes from #3836 and was already validated.

### :link: Related

- #3836

### CODE REVIEW

**Comment by antleblanc** [comment]
Could you please sign commit? (`310dfaa`)
Could you also provide ticket reference if any (both body commit and Pull Request description)

Thank you!

File: packages/manager/modules/dbaas-logs/src/logs/empty/empty.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
+<div class="container mx-0 px-0 ldp-project-empty">
+    <div class="row align-items-center">
+        <div class="col-md-4">
+            <div data-ng-class="{ 'placeholder p-5' : !$ctrl.imageSource }">
+                <img
+                    data-ng-src="{{:: $ctrl.imageSource || $ctrl.placeholder }}"
+                    alt="{{::'alt_illustration' | translate}}"
```
Comment (antleblanc) [inline packages/manager/modules/dbaas-logs/src/logs/empty/empty.html:7]: Since translation is empty then you can leave the attribute empty.

```suggestion
                    alt=""
```

File: packages/manager/modules/dbaas-logs/src/logs/empty/index.less [hunk 2]
```diff
@@ -0,0 +1,31 @@
+.ldp-project-empty {
```
Comment (antleblanc) [inline packages/manager/modules/dbaas-logs/src/logs/empty/index.less:1]: **FYI**: we tend to avoid using custom style.

**thought:**  Having custom style mean probably that you miss some functionalities from [`@ovh-ux/ui-kit`](https://ovh.github.io/ovh-ui-kit/) package.
If it's the case, please don't hesitate to raise an Issue.

It can be considered on a separate Pull Request (non-blocking).

File: packages/manager/modules/dbaas-logs/src/logs/empty/translations/Messages_fr_FR.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
+  "ldp_project_empty_more": "En savoir plus",
+  "ldp_project_empty_new_tab": "Nouvel onglet",
+  "alt_illustration": ""
```
Comment (antleblanc) [inline packages/manager/modules/dbaas-logs/src/logs/empty/translations/Messages_fr_FR.json:4]: Can be removed.
```suggestion
```

File: packages/manager/modules/dbaas-logs/src/logs/logs.scss [hunk 4]
```diff
@@ -5,3 +5,8 @@ dbaas-logs {
   @import 'bootstrap4/scss/_utilities.scss';
   @import 'bootstrap4/scss/_grid.scss';
 }
+
+#order-messages-anchor {
```
Comment (antleblanc) [inline packages/manager/modules/dbaas-logs/src/logs/logs.scss:9]: Is it required?

File: packages/manager/modules/dbaas-logs/src/logs/order/logs-order.html [hunk 5]
```diff
@@ -0,0 +1,127 @@
+<div class="row p-4">
+    <div class="col-lg-9 p-4">
+        <h1 data-translate="logs_order_title" class="m-0"></h1>
```
Comment (antleblanc) [inline packages/manager/modules/dbaas-logs/src/logs/order/logs-order.html:3]: Why not considering using [`<oui-header`](https://ovh.github.io/ovh-ui-kit/?path=/story/design-system-components-header-webcomponent--default)?

File: packages/manager/modules/dbaas-logs/src/logs/logs.scss (hunk 4 above)
Comment (Heiiji) [reply packages/manager/modules/dbaas-logs/src/logs/logs.scss:9]: nope

File: packages/manager/modules/dbaas-logs/src/logs/empty/index.less (hunk 2 above)
Comment (Heiiji) [reply packages/manager/modules/dbaas-logs/src/logs/empty/index.less:1]: this empty state can be replaced soon with another PR with a standalone empty state. (https://github.com/ovh/manager/pull/3846)

File: packages/manager/modules/dbaas-logs/src/logs/empty/empty.html [hunk 6]
```diff
@@ -0,0 +1,52 @@
+<div class="container mx-0 px-0 ldp-project-empty">
+    <div class="row align-items-center">
+        <div class="col-md-4">
+            <div data-ng-class="{ 'placeholder p-5' : !$ctrl.imageSource }">
+                <img
+                    data-ng-src="{{:: $ctrl.imageSource || $ctrl.placeholder }}"
+                    alt=""
```
Comment (JeremyDec) [inline packages/manager/modules/dbaas-logs/src/logs/empty/empty.html:7]: ?
```suggestion
                    alt="{{$ctrl.imageAlt}}" 
```

Comment (marie-j) [reply packages/manager/modules/dbaas-logs/src/logs/empty/empty.html:7]: If the image has only some decorative purpose and don't provide further information, it's ok to leave an empty alt (as long as it's there)

Comment (Heiiji) [reply packages/manager/modules/dbaas-logs/src/logs/empty/empty.html:7]: yes, it has been removed

**Review by antleblanc — APPROVED** [review]
